src/components/MultiMatrix/MultiMatrix.vue

```diff
@@ -635,7 +742,23 @@ export default Vue.extend({
         .attr('width', this.cellSize - 2)
         .attr('height', this.cellSize - 2)
         .attr('rx', cellRadius)
-        .style('fill', (d: Cell) => this.colorScale(d.z))
+        .style('fill', (d: Cell) => {
+          if (d.rowCellType === undefined) {
+            return this.colorScale(d.z);
+          }
+          if (d.rowCellType === 'supernode' && d.colCellType === 'supernode') {
+            return this.aggrColorScale(d.z);
+          }
+          if (d.rowCellType === 'childnode' && d.colCellType === 'childnode') {
+            return this.childColorScale(d.z);
+          }
+          if (d.rowCellType === 'childnode' && d.colCellType === 'supernode') {
+            return this.childColorScale(d.z);
+          }
+          if (d.rowCellType === 'supernode' && d.colCellType === 'childnode') {
+            return this.childColorScale(d.z);
+          }
```

Comment:
Another spot to simplify with an OR
```suggestion
          if (d.rowCellType === 'childnode' || d.colCellType === 'childnode') {
            return this.childColorScale(d.z);
          }
```